test/plugin-multi-tenant/int.spec.ts

```diff
@@ -48,5 +49,91 @@ describe('@payloadcms/plugin-multi-tenant', () => {
 
       expect(tenant1).toHaveProperty('id')
     })
+
+    describe('relationships', () => {
+      let anchorBarRelationships: PaginatedDocs<Relationship>
+      let blueDogRelationships: PaginatedDocs<Relationship>
+      let anchorBarTenantID: DefaultDocumentIDType
+      let blueDogTenantID: DefaultDocumentIDType
+
+      beforeEach(async () => {
+        anchorBarRelationships = await payload.find({
+          collection: 'relationships',
+          where: {
+            'tenant.name': {
+              equals: 'Anchor Bar',
+            },
+          },
+        })
+
+        blueDogRelationships = await payload.find({
+          collection: 'relationships',
+          where: {
+            'tenant.name': {
+              equals: 'Blue Dog',
+            },
+          },
+        })
+
+        // @ts-expect-error unsafe access okay in test
+
+        anchorBarTenantID = anchorBarRelationships.docs[0].tenant.id
+        // @ts-expect-error unsafe access okay in test
+        blueDogTenantID = blueDogRelationships.docs[0].tenant.id
+      })
+
+      it('ensure relationship document with relationship within same tenant can be created', async () => {
+        const newRelationship = await payload.create({
+          collection: 'relationships',
+          data: {
+            title: 'Relationship to Anchor Bar',
+            // @ts-expect-error unsafe access okay in test
+            relationship: anchorBarRelationships.docs[0].id,
+            tenant: anchorBarTenantID,
+          },
+          req: {
+            // Relationship won't be filtered by tenant based on the tenant of document, but the tenant of the cookie
```

Comment:
I'm a little confused by these comments. The document will be filtered by the tenant in data not the cookie, right?